applications/ULFapplication/Lagrangian-droplet_test/ProjectParameters.py

```diff
@@ -0,0 +1,89 @@
+domain_size = 2
```

Comment:
I guess that your application still works with the old ProjectParameters Python scripts. About a year ago, we decided to use the .json file format for the ProjectParameters, meaning that the old ProjectParameters file format became deprecated. It would be a good idea to take the chance to update that.

Reply:
At the moment I disagree with this since it would require changing the whole ULF application. ULF is not working with the same problem type as the one people of FluidDynamicsApplication are using. Our problemtype write the input parameters files in the old format. We can discuss passing to the new format, but I believe this is not necessary for this pull.